web/server/www/scripts/codecheckerviewer/CommentView.js

```diff
@@ -209,15 +250,19 @@ function (declare, dom, style, topic, Memory, Observable, ConfirmDialog,
         } catch (ex) { util.handleThriftException(ex); }
 
         comments.forEach(function (comment) {
-          that._comments.addChild(new Comment({
-            class    : 'comment',
-            reportId : reportId,
-            cId      : comment.id,
-            author   : comment.author,
-            time     : comment.createdAt,
-            message  : comment.message,
-            sender   : sender
-          }));
+          if (comment.kind === CC_OBJECTS.CommentKind.SYSTEM) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['CC_OBJECTS' is not defined. (no-undef)](https://app.codacy.com/app/CodeChecker/codechecker/pullRequest?prid=3374942)